nin/backend/utils.js

```diff
@@ -10,10 +10,10 @@ function findProjectRoot(currentPath) {
         manifest = path.join(base, 'nin.json');
 
     if (fs.existsSync(manifest)) {
-      return base;
+      return base.replace(/\/$/, '');
```

Comment:
If that were the case, this shouldn't work either, I believe: https://github.com/ninjadev/nin/pull/217/files#diff-ff41c28559292673d9de3d8372834e69R15